products/idn/docs/identity-now/global-platform-resources/index.md

```diff
@@ -0,0 +1,21 @@
+---
+id: global-platform-resources
+title: Global Platform Resources
+pagination_label: Global Platform Resources
+sidebar_label: Global Platform Resources
+sidebar_position: 1
+sidebar_class_name: globalResources
+keywords: ['IdentityNow', 'resources', 'configuration']
+description: All the documentation that applies to the entire IdentityNow application
```

Comment:
Can you change this to "Global platform resources are useful for all IdentityNow platforms."